catalog/spatial/csw/spatial-csw-source/pom.xml

```diff
@@ -207,22 +213,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.62</minimum>
+                                            <minimum>0.0</minimum>
```

Comment:
So right now all the functionality is in the abstract class so I think it makes sense to have the test with it. I agree that the concrete implementations should just be able to test their new/additional functionality but right now there is none. To vina's question the coverage went to the spatial-csw-source-common and there is really nothing to test in the concrete implementation. All that to say I don't know that I want to move the test back into spacial-csw-source. If you still want a test in spatial-csw-source let me know and we can work something out.